src/mudlet-lua/lua/StringUtils.lua

```diff
@@ -63,6 +63,7 @@ end
 
 --- Documentation: https://wiki.mudlet.org/w/Manual:String_Functions#string.split
 function string:split(delimiter)
+  delimiter = delimiter or " "
```

Comment:
Can you document this after merging?

Reply:
went ahead and documented it as being defaulted to " " in 4.7+